tests/test_dag_extension.py

```diff
@@ -0,0 +1,104 @@
+from marquez.airflow import MarquezDag
+from unittest.mock import Mock, patch, create_autospec
+from datetime import datetime
+from airflow.utils.state import State
+import airflow.models
+import pytest
+from croniter import croniter
+import pendulum
+import marquez.utils
+from marquez_client.marquez import MarquezClient
```

Comment:
please use `isort` to order the imports. I have a jira for a build step that will eventually run/require `isort`

Reply:
I think the convention is to call the directory `test` instead of `tests`.

Reply:
sounds good!